src/tests/ftest/container/fill_destroy_loop.py

```diff
@@ -85,8 +85,9 @@ def test_fill_destroy_cont_loop(self):
         :avocado: tags=fill_cont_pool_stress,test_fill_destroy_cont_loop
         """
         testloop = self.params.get("testloop", "/run/pool/*")
-        # create pool
+        # create pool and enable the aggregation
         self.add_pool()
+        self.pool.set_property("reclaim", "time")
```

Comment:
I think aggregation is enabled by default (maybe I'm wrong). But also, my previous comment was about whether or not we need to wait for aggregation to start before checking `if free_space_after <= free_space_before:` to avoid intermittent failures. (again, I might be wrong about this)